woocommerce/wc-api-dev
# Fixes 'single_select_page' settings not showing up in responses.

Right now settings with the type 'single_select_page' are not showing up in the settings response. For example `woocommerce_shop_page_id`.

This fixes that by supporting the setting type. It includes a test to look at `woocommerce_shop_page_id`.

To Test:
* Run `phpunit`. You can verify this test specifically with `phpunit --filter=test_woocommerce_single_select_page_setting`.
* Make a request (using something like Postman) to `/wc/v3/settings/products/woocommerce_shop_page_id`. Verify you see a setting. Options should contain an array of pages on your site.
